src/components/organisms/SfTabMenu/SfTab/SfTab.js

```diff
@@ -0,0 +1,21 @@
+export const TAB_COMPONENT_NAME = "SfTab";
+
+export default {
+  name: TAB_COMPONENT_NAME,
+  inject: ["sharedState"],
```

Comment:
use this.$parent or (even better) explict props from slot scope

Reply:
without provide/inject?

Reply:
injectors are just harder for people to use than scoped slots. If one wants to have custom nav i guess we shouldn't force them to use injectors

Reply:
Nvmd, i rethinked and it's fine